### PR TITLE
[YOLO] code clean up and optimization

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Refactor some repeating calls  into a one function.

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

File: custom_model_runner/datarobot_drum/resource/predict_mixin.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def _check_mimetype_support(self, mimetype):
             return {"message": error_message}, HTTP_422_UNPROCESSABLE_ENTITY
         return None
 
-    def _predict(self, logger=None):
+    def _do_predict(self, logger=None):
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/resource/predict_mixin.py:110]: having two functions named do_predict and _do_predict seems confusing. Can you give either of them a more descriptive name?

Comment (yakov-g) [reply custom_model_runner/datarobot_drum/resource/predict_mixin.py]: Generally I don't see this confusing as `some_func` is usually a public wrapper to `_some_func`.
But here I tried to rename as we had `do_predict` in BasePredictor as well.

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

File: custom_model_runner/datarobot_drum/resource/predict_mixin.py (hunk 1 above)
Comment (rvorobii) [reply custom_model_runner/datarobot_drum/resource/predict_mixin.py]: to me, `do_predict` + `_do_predict` is much better than `do_predict` + `do_predict_predictor`. method name is super confusing.

`some_func` + `_some_func` is a common pattern.

**Review by timsetsfire — APPROVED** [review]

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Pull request merged**